servers/src/main/java/tachyon/master/block/BlockMaster.java

```diff
@@ -258,9 +258,9 @@ public long getUsedBytes() {
    *
    * @return a list of worker info
    */
-  public List<WorkerInfo> getLostWorkersInfo() {
+  public Set<WorkerInfo> getLostWorkersInfo() {
```

Comment:
There's no specific semantics on the order of the {{WorkersInfo}}s returned here. I believe it is better to return a set instead.